apps/frontend/lib/tools/credit-report/credit-report-schema.js

```diff
@@ -0,0 +1,214 @@
+        properties: {
+          debtAmount: {
+            $format: "currency",
+            title: "Amount",
+            type: "number",
+          },
+          debtType: {
+            $ref: "#/definitions/debt-types",
+            title: "Debt Type",
+            type: "string",
+          },
+        },
+        required: ["debtType", "debtAmount"],
+        title: "Amount of Debt Disputed",
+        type: "object",
+      },
+      personalInformation: {
+        properties: {
+          address: {
+            title: "Your Mailing Address",
+            type: "string",
+          },
+          agencies: {
+            description: "Reporting agencies your dispute will be sent to",
+            items: {
+              enum: ["Experian", "Equifax", "TransUnion"],
+              type: "string",
+            },
+            minItems: 1,
+            title: "Reporting agencies",
+            type: "array",
+            uniqueItems: true,
+          },
+          city: {
+            title: "Your City",
+            type: "string",
+          },
+          currentCreditor: {
+            title: "Current Creditor",
+            type: "string",
+          },
+          dob: {
+            format: "date",
+            title: "Your date of birth?",
+            type: "string",
+          },
+          email: {
+            format: "email",
+            title: "Your email",
+            type: "string",
+          },
+          name: {
+            title: "Your Full Name",
+            type: "string",
+          },
+          originalCreditor: {
+            title: "Original Creditor",
+            type: "string",
+          },
+          phone: {
+            $format: "telephone",
+            title: "Your telephone",
+            type: "number",
+          },
+          ssn: {
+            $format: "ssn",
+            title: "Your SSN",
+            type: "number",
+          },
+          state: {
+            $ref: "#/definitions/usa-states",
+            title: "Your State",
+          },
+          "zip-code": {
+            pattern: "[0-9]{5}",
+            title: "Your Zip Code",
+            type: "string",
+          },
+        },
+        required: [
+          "address",
+          "agencies",
+          "city",
+          "currentCreditor",
+          "dob",
+          "email",
+          "name",
+          "originalCreditor",
+          "phone",
+          "ssn",
+          "state",
+          "zip-code",
+        ],
+        title: "Personal Information",
+        type: "object",
+      },
+    },
+    title: "Dispute Any Debt in Collections",
+    type: "object",
+  },
+  ui: {
+    debts: {
+      debtAmount: {
+        classNames: "prefix-currency",
+      },
+      debtType: {
+        "ui:help": "If you don't see your type of debt, choose 'Other'",
+        "ui:placeholder": "Select one",
+      },
+    },
+    personalInformation: {
```

Comment:
You need to specify:

```
personalInformation: {
    agency: {
         "ui:widget": "checkboxes"
    }
}
```

Take a look at examples within https://mozilla-services.github.io/react-jsonschema-form/ especially "Arrays" to see about this case. Our previous implementation at https://github.com/debtcollective/disputes/blob/master/apps/frontend/pages/disputes/tax/e.js#L31 with https://github.com/debtcollective/disputes/blob/master/apps/frontend/pages/disputes/tax/e.js#L79